addon_service/admin/__init__.py

```diff
@@ -23,9 +23,6 @@ class ExternalStorageServiceAdmin(GravyvaletModelAdmin):
         "int_addon_imp": known_imps.AddonImpNumbers,
         "int_credentials_format": CredentialsFormats,
         "int_service_type": ServiceTypes,
-    }
-
-    enum_multiple_choice_fields = {
```

Comment:
Should we be getting rid of this, or does it behave differently in admin vs choice fields? It looks like it gives different controls compared to the enum_choice_fields. https://github.com/CenterForOpenScience/gravyvalet/blob/c197c7e6a81eb340e480733299d912b82dd07880/addon_service/admin/_base.py#L69-L79

Reply:
good point, I will remove these changes